nexus/tests/unit/test_pyscf_input.py

```diff
@@ -554,10 +554,23 @@ def test_write():
         
         ### generated conversion text ###
         from PyscfToQmcpack import savetoqmcpack
-        savetoqmcpack(cell,mf,'scf',kpts)
+        tiling = [2,1,1]
+        sp_kpoints = array([
+            [0.0, 0.0, 0.0]])
+        sp_kmap    = array([
+            [0, 1]])
+        for n,kp in enumerate(sp_kpoints):
+            savetoqmcpack(cell,mf,'scf.twistnum_{{}}'.format(str(n).zfill(3)),kmesh=tiling,kpts=kpts[sp_kmap[n]],sp_twist=kp)
+        #end for
```

Comment:
In order for this to work with kpoints in a ncsf style, we need to also pass the sp_kmap. The reason is that when running N kpts for example, mf method has N mo_energies and N mo_coeffs. but the tiling will only set a subset of kpts. we need then to use the index of the kptks provided by the sp_kmap. 

 savetoqmcpack(cell,mf,'scf.twistnum_{{}}'.format(str(n).zfill(3)),kmesh=tiling,kpts=kpts[sp_kmap[n]],sp_twist=kp,kmap=sp_kmap[n])